PlaybackLab/BasicPlaybackWithScan/app/build.gradle

```diff
@@ -19,7 +19,7 @@ android {
 }
 
 task copyTask(type: Copy) {
-    from new File(projectDir, '../../vendor/Ooyala/OoyalaSDK-Android/OoyalaSDK.jar')
+    from new File(projectDir, '../../../vendor/Ooyala/OoyalaSDK-Android/OoyalaSDK.jar')
```

Comment:
Is there a root directory we can use instead of ..'s so we don't have to keep track of directory structure?